app/src/main/java/com/sdp/movemeet/utility/ActivitiesUpdater.java

```diff
@@ -29,16 +36,18 @@
     private FirebaseFirestore db;
     private ArrayList<Activity> activities;
 
-    private static final String TAG = "Activities updater TAG";
+    //private ActivityCache cache;
+    private MainMapFragment mapFragment;
 
-    public static final int MAX_NUMBER_ACTIVITIES_TO_DISPLAY = 10;
```

Comment:
By removing this, does it mean that all activities will now be displayed? And that we are no more limited to 10 activities?

Reply:
Indeed. I  put this first because the updater was not optimized, and had a ton of duplicates, so it made the app really slow at some point. But now that the updater is optimized, I don't see why we would need to have a max number of activities to display. But to be sure, we'll still have to test with A LOT of activities in the DB, we might want to first use the distance calculator to only display a certain number of activities, to prevent having the map loading for a few minutes.

Reply:
Ok I understand, thank you!